packages/fxa-payments-server/src/components/PaymentForm/index.scss

```diff
@@ -4,28 +4,38 @@ form.payment {
     color: #686869;
     font-size: 12px;
     font-weight: 500;
+    line-height: 16px;
     text-align: center;
+
+    p {
+      margin-bottom: 4px;
+      margin-top: 0;
+    }
+
+    p:last-child {
+      margin-bottom: 0;
+    }
   }
 
   .input-row {
     input {
-      font-size: 13px;
+      font-size: 14px;
     }
 
     input::placeholder {
-      color: rgba(12, 12, 13, 0.54) !important;
-      font-size: 13px;
-      font-weight: 500;
+      color: #767676 !important;
+      font-size: 14px;
+      font-weight: 400;
       height: 45px;
       line-height: 45px;
     }
   }
 }
 
-@media (min-width: 320px) and (max-width: 480px) {
+@media (max-width: 520px) {
```

Comment:
You should be able to replace this line with `@include respond-to('simpleSmall') {`.